### DISCUSSION TITLE
Problem in authentication

### DISCUSSION THREAD

**Discussion by lefterisemm** [etsy/open-api#1252] [Q&A]
Good evening,
I am trying to connect an erp system to https://www.etsy.com/shop/DimitrasWorkshop to download the orders to my system.
I'm trying to authenticate with the following instructions https://developers.etsy.com/documentation/tutorials/quickstart/#initial-setup but I keep getting error messages.
Can someone help me with this part and give me directions on how to get the orders?
Thanks

**Comment by slider1234** [comment]
What programming language are you working with and what are your errors?

**Comment by lefterisemm** [comment]
i'm using javascript and the error i'm getting is 'cannot get'.
